README.md

```diff
@@ -1,5 +1,7 @@
 # Google Student Training in Engineering Program
 
+This repo contains Chris Elliott's portfolio and SPS projects.
+
 This repo contains the projects you'll work on for the first half of STEP.
 This work is based on the [Google Software Product Sprint](https://g.co/softwareproductsprint) program.
```

Comment:
No need to keep this extra text in here now that you've added your new description. You also don't need the header anymore.